fala/templates/adviser/single_category_search.html

```diff
@@ -0,0 +1,125 @@
+{% extends 'adviser/adviser_base.html' %}
+
+{% block pageTitle %}
+  {% if results %}
+    Search results
+  {% else %}
+    Find a Legal Aid Adviser or Family Mediator
+  {% endif %}
+{% endblock %}
+
+{% import "macros/element.html" as Element %}
+{% import "macros/forms.html" as Form %}
+
+{%- from 'govuk_frontend_jinja/components/error-summary/macro.html' import govukErrorSummary %}
+{%- from 'govuk_frontend_jinja/components/checkboxes/macro.html' import govukCheckboxes %}
+{%- from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
+
+{% block content %}
+  {% if results %}
+    <!-- Content for search results -->
```

Comment:
Question - why aren't we using the `results.html`? Is that part of another ticket

Reply:
i just wanted to get some results on display, we can write a ticket to make sure results template is used by this search I was just trying to go with the most simplified results page I could to display the data I was getting bacl